src/scripts/components/story-header/story-header.styl

```diff
@@ -0,0 +1,23 @@
+@require '../../../variables.styl'
+
+.storyHeader
+  display: flex
+  flex-shrink: 0
+  justify-content: space-between
+  align-items: center
+  height: emCalc(72px)
```

Comment:
Put in variables file, or do you still plan to make a general header component? Would probably be the best solution